todocore/pom.xml

```diff
@@ -31,6 +31,12 @@
             <artifactId>jjwt</artifactId>
             <version>${jsonwebtoken.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>1.15.2</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This dependency should be in the higher level pom.xml file that we have in the project, check it please.